src/Reflection/Apply.agda

```diff
@@ -0,0 +1,200 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Function application of reflected terms
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Data.Nat
+
+module Reflection.Apply (limit : ℕ) where
+
+open import Level renaming (suc to lsuc; _⊔_ to lmax)
```

Comment:
Hmm `_⊔_ to lmax` isn't a standard renaming. Any chance we could keep `_⊔_`?

Reply:
Another bit of convention from cubical. I changed it to lmax because it was clashing with the same name defined in Data.Nat. I guess I can just hide the definition from Data.Nat since I don't use it.